docs/UserGuide.md

```diff
@@ -300,8 +307,9 @@ Action | Format, Examples
 
 Term used | Meaning
 --------|------------------
-Pool | A group of employees carpooling together. Consists of one driver and at least one passenger.
+Pool | A group of employees carpooling together. Consists of one driver and at least one passenger. The pools generated for a specifc day and time are the same every week unless reorganised by the user. 
 Passenger | An employee carpooling with at least one driver.
-TripDay | Day of intended carpooling trip 
+TripDay | Day of the intended carpooling trip.  
 TripTime | Time of intended carpooling trip 
```

Comment:
Full stop for TripDay but none for TripTime, and also TripTime now doesnt have an extra 'the'